src/styles/styles.js

```diff
@@ -1185,14 +1185,11 @@ const styles = {
         fontFamily: fontFamily.GTA,
         height: 20,
         lineHeight: 20,
-        ...whiteSpace.noWrap,
-    },
-
-    optionDisplayNameCompact: {
```

Comment:
Not sure I understand this change. Are we removing this style? This is used at two places (`OptionRowLHN`, and also `OptionRow`). I think removing this will break the styling in OptionRow.

Reply:
Sorry, I missed that it is also being used at OptionRow component. I merge both styles to fix the #announce text got truncated on not compact mode. I will fix it later. Thanks for pointing it out!

Reply:
Done